apps/src/code-studio/youTubeAvailabilityEndpointURL.js

```diff
@@ -0,0 +1,14 @@
+export default function youTubeAvailabilityEndpointURL(noCookie) {
```

Comment:
nit: mind adding a quick doc on what this is meant to be used for?

Reply:
Done.